app/cli/cmd/attestation_init.go

```diff
@@ -77,7 +77,7 @@ func newAttestationInitCmd() *cobra.Command {
 				return newGracefulError(err)
 			}
 
-			return encodeOutput(res, attestationStatusTableOutput)
```

Comment:
another way of doing this a little bit more declarative could have been to have two aliases.

```go
func emptyStatusTable(result *action.AttestationStatusResult) {
attestationStatusTableOutput)(result, false)
}


func fullStatusTable(result *action.AttestationStatusResult) {
attestationStatusTableOutput)(result, true)
}
```

Then you could keep the return similar `return encodeOutput(res, emptyStatusTable)`